celery/backends/cache.py

```diff
@@ -149,3 +149,12 @@ def __reduce__(self, args=(), kwargs={}):
                  expires=self.expires,
                  options=self.options))
         return super(CacheBackend, self).__reduce__(args, kwargs)
+
+    def as_uri(self, *args, **kwargs):
+        """
+        Return the backend as an URI. It properly handles the
+        case of multiple servers. It doesn't try to sanitize
+        password because memcached URIs doesn't support them.
```

Comment:
Yeah, I think you can technically using SASL but the Celery cache backend doesn't let you do so via the URL